src/generic/processing-notification/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Badge, Icon } from '@edx/paragon';
+import { Settings as IconSettings } from '@edx/paragon/icons';
+
```

Comment:
do we need to create generic store for showing/hiding this component?

Reply:
I think not at the moment. As separate components, we have `AlertMessage`, `InternetConnectionAlert` components that also appear when any state changes. For them, we do not have the logic of a separate store for any actions related to their behavior.

What do you think?

Reply:
lets do it next your task with applying component to the page